lib/bowling_score.rb

```diff
@@ -0,0 +1,50 @@
+class BowlingScore
```

Comment:
All your method names and variable names are clear and explain what they contain or what they should do.